.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
The first advantage that I can remember is that you can share the same configuration for other tools like `babel-cli` for instance. With `babel-cli` we can use babel es6 with all javascript files, node files include.
Something very useful with `babel-cli` is using Object spread operator `{...}` to merge multiple node configuration files (webpack/gulp).
And we can have only one eslint rules for all js files.